src/etc/inc/auth.inc

```diff
@@ -488,7 +488,7 @@ function local_user_del($user)
 
 function local_user_set_password(&$user, $password)
 {
-    $user['password'] = crypt($password, '$6$');
+    $user['password'] = generate_password_hash($password, 10);
```

Comment:
wrote a little benchmark tool to test it on my laptop (Linux on an i7-2620M) using the bcrypt gem:

100 times, cost=10, delta t=6.92085047, mean=0.0692085047
100 times, cost=12, delta t=27.533339248, mean=0.27533339248